astroquery/mast/tests/test_mast_remote.py

```diff
@@ -294,6 +294,38 @@ def test_observations_download_file(self, tmpdir):
         result = mast.Observations.download_file(uri)
         assert result == ('COMPLETE', None, None)
 
+    def test_get_cloud_uri(self):
+        test_obs_id = '25568122'
+
+        # get a product list
+        product = mast.Observations.get_product_list(test_obs_id)[24]
+
+        assert len(product) > 0, f'No product found for OBSID {test_obs_id}. Unable to move forward with getting URIs from the cloud.'
+
+        # enable access to public AWS S3 bucket
+        mast.Observations.enable_cloud_dataset()
+
+        # get uri
+        uri = mast.Observations.get_cloud_uri(product)
+
+        assert len(uri) > 0, f'Product for OBSID {test_obs_id} was not found in the cloud.'
+
+    def test_get_cloud_uris(self):
+        test_obs_id = '25568122'
+
+        # get a product list
+        products = mast.Observations.get_product_list(test_obs_id)[24:]
+
+        assert len(products) > 0, f'No products found for OBSID {test_obs_id}. Unable to move forward with getting URIs from the cloud.'
```

Comment:
We don't necessarily write assert messages in the tests, but you're right, they likely don't hurt to have :)